src/gui/Components/ScreenElements/Table/PositionedField.module.css

```
@@ -3,7 +3,7 @@ Copyright 2005 - 2021 Advantage Solutions, s. r. o.
 
 This file is part of ORIGAM (http://www.origam.org).
 
-ORIGAM is free software: you can redistribute it and/or modify
+ORIGAM is free software: you canvar(--red)istribute it and/or modify
```

Big LMAO on this...
🤣🤣🤣🤣🤣🤣🤣

You might like to tune up your regexes better :)

Please fix similar issues if happened elsewhere.

```suggestion
ORIGAM is free software: you can redistribute it and/or modify
```